ruby_on_rails/aws.md

```diff
@@ -25,6 +25,7 @@ Make sure `renuo -v` shows the [newest version](https://github.com/renuo/renuo-c
 
 The following command will create command-line-commands to set up S3 and CloudFlare. Before that, the command makes sure,
 you have the aws-profile and the aws-cli set up, so you have the necessary rights to run the commands.
+If you don't, follow the [AWS CLI Setup Guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
```

Comment:
maybe add that the `aws_access_key_id` and `aws_secret_access_key` are stored in a note attached to the admin@renuo.ch credentials for aws on passwork.me

Reply:
since this project is open source I would not